val.py

```diff
@@ -175,10 +195,12 @@ def run(
         callbacks=Callbacks(),
         compute_loss=None,
         tagged_data=False,
-        skip_evaluation=False,
-        save_blurred_image=False):
+        skip_evaluation=True,
+        save_blurred_image=False,
+        customer_name=""):
```

Comment:
Do we want a default customer_name or shall we fail if not provided?

Reply:
example_customer is the deafult, provided by parse_opt function